evergreen/run-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -o errexit  # Exit the script with error if any of the commands fail
+
+MONGODB_URI=${MONGODB_URI:=mongodb://localhost:27017/}
+
+dotnet clean "./MongoDB.AspNetCore.OData.sln"
+for TEST_PROJECT in ./tests/**/*.Tests.csproj; do
+  ## Evergreen require JunitXml test results format. Have to add logger as the dependency for test project
```

Comment:
```suggestion
  ## Evergreen requires JunitXml test results format. Add JunitXml logger as the dependency to the test projects
```

Reply:
Not relevant anymore, as we moved this dependency into project.